IN_SOPT_Second_Task/IN_SOPT_Second_Task/AddSubViews.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  AddSubViews.swift
+//  IN_SOPT_First_Task
+//
+//  Created by 박의서 on 2022/10/09.
+//
+
+import Foundation
+
+import UIKit
```

Comment:
UIKit가 Foundation을 포함하기 때문에 
```suggestion
import UIKit
```
만 작성해도 될 것 같아요!

(+ 추가로 프레임워크간의 관계도 알아보면 좋겠져~~)